packages/components/src/Filters/FilterInput.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { Checkbox, Radio } from '@patternfly/react-core';
+
+export interface FilterInputProps {
+  addRemoveFilters?: (value?: string, param?: string, type?: string, checked?: boolean) => void;
+  className?: string;
+  currentPage?: string;
+  id: string;
+  label: React.ReactNode;
+  param: string;
+  value?: string;
+  filters?: Record<string, string>;
+  type?: string;
+  onChange?: (checked: boolean, event: React.FormEvent<HTMLInputElement>) => void;
```

Comment:
`onChange` is not present in the original prop types definition.

Reply:
It's a bit tricky but we can extend some general HTMLElement prop so we don't have to include the `onChange` prop. How about `HTMLElement`?